activerecord/lib/active_record/fixtures.rb

```diff
@@ -761,13 +773,25 @@ def column_names
         @column_names ||= @connection.columns(@table_name).collect(&:name)
       end
 
-      def read_fixture_files(path, model_class)
+      def model_class=(class_name)
```

Comment:
Do we still need this extraction? It looks like we could revert that part back to how it's working on `master`?